cpu/efr32mg1p/Makefile.include

```diff
@@ -0,0 +1,15 @@
+export CPU_ARCH = cortex-m4
+export CPU_FAM = efr32mg1p
+
+# enable memory protection unit
+USEMODULE += mpu_stack_guard
+USEMODULE += pm_layered
```

Comment:
shouldn't `pm_layered` be included from `efm32_common`?

Reply:
For the STM32F1/F2/F4 it is also on per-CPU base. Any preference?

Reply:
The STMs do this on a per-CPU base, as we simply didn't touch all of the CPUs, yet. In the end this will be done in `stm32_common`. So as I also see all EFM32 CPUs use that module, I would put the usemodule define there.

Reply:
Done.